### DISCUSSION TITLE
Consider collaborating with alex / retext-equality

### DISCUSSION THREAD

**Discussion by smokris** [inclusivenaming/org#14] [Ideas]
https://alexjs.com/ is a text linter designed to catch insensitive/inconsiderate writing.  They've already built an extensive, parseable list of [problematic words, suggestions for alternatives](https://github.com/retextjs/retext-equality/blob/main/rules.md#list-of-rules), including many citations ([e.g.](https://github.com/retextjs/retext-equality/blob/main/data/en/race.yml#L63)).

Since the Inclusive Naming Initiative is looking at building language recommendations, it might be useful to join forces with _alex_ to reduce redundant work.

**Comment by quaid** [comment]
I think we missed this idea in the end of the year flow, but it came up from another direction in a chat channel, and is now tracked in this issue:

https://github.com/inclusivenaming/org/issues/57
